src/visitor.rs

```diff
@@ -25,20 +27,73 @@ impl<'a> Visitor<'a> for ImportVisitor {
         match stmt {
             Stmt::Import(import_stmt) => {
```

Comment:
I think the additions make this code a bit long, and therefore hard to read; would it be possible to make it more modular? e.g. something like

```rs
match stmt {
    Stmt::Import(import_stmt) => self.handle_import(import_stmt),
    Stmt::ImportFrom(import_from_stmt) => self.handle_import_from(import_from_stmt),
    ...
```

Reply:
Would it be possible to add one or two small comments/docstrings to `visitor.rs`? e.g.

- a docstring under `ImportVisitor ` that explains what `import_module_names` is, 
- one or two sentences about what the code under `Stmt::Expr(expr_stmt) => {` does? 

While both are definitely clear in the context of this PR, I'm afraid in the future it might be less apparent at first glance.

Reply:
- [ ] TODO: refactor long Stmt::Import and Stmt::ImportFrom arms of match statement
- [x] <s>TODO</s> docstring for ImportVisitor explaining what import_module_names is 09439b7
- [ ] TODO: document Stmt::Expr match arm